Hunting Queries/OfficeActivity/OfficeMailForwarding_hunting.txt

```diff
@@ -9,10 +9,11 @@
 // Techniques: #Exfiltration
 //
 OfficeActivity
-| where Operation == "Set-Mailbox"
+| where (Operation == "Set-Mailbox" and Parameters contains 'ForwardingSmtpAddress') 
+    or (Operation == 'New-InboxRule' and Parameters contains 'ForwardTo')
 | where TimeGenerated >= ago(30d)
```

Comment:
Only thought is to put the Time clause above the where clause.  This could help Perf on a large data set.

Reply:
Good point, change made.